### DISCUSSION TITLE
Add a dedicated, comprehensive "Permission levels" page

### DISCUSSION THREAD

**Discussion by ivanaur** [semaphoreci/semaphore#66] [Documentation improvements]
Roles/permission levels exist on several levels within the organization. They also exist on multiple pages in the Semaphore docs:

1. [Organization role](https://storage.googleapis.com/sem-docs/main/index.html#/docs/using-semaphore/organizations#roles)
2. [Group role](https://storage.googleapis.com/sem-docs/main/index.html#/docs/using-semaphore/organizations#group-role)
3. [Project role](https://storage.googleapis.com/sem-docs/main/index.html#/docs/using-semaphore/projects#people-roles)

What should be added:

- How does each role level correlate to their roles on GitHub (eg. does "Organization Admin" on Semaphore overwrite the "Read" permission level on GitHub)
- Expand on "[everything](https://storage.googleapis.com/sem-docs/main/index.html#/docs/using-semaphore/organizations#roles:~:text=Can%20do-,everything,-within%20the%20organization)" ie. how do roles correlate with different features such as "Secrets", "Artifacts", etc.
- Can Organization role overwrite the Group role or Project role and vice versa

**Comment by TomFern** [comment]
@ivanaur good idea! I created an issue to track it: https://github.com/semaphoreci/semaphore/issues/87
